src/rpc/net.cpp

```diff
@@ -702,9 +702,7 @@ static RPCHelpMan setban()
         throw std::runtime_error(help.ToString());
     }
     NodeContext& node = EnsureAnyNodeContext(request.context);
```

Comment:
why not use EnsureAnyBanman here?

Reply:
Because we need `connman` to disconnect the node right after banning.

Reply:
e.g:
```cpp
banman.Ban(subNet, banTime, absolute);
if (node.connman) {
    node.connman->DisconnectNode(subNet);
}
```